content/_data/changelogs/weekly.yml

```diff
@@ -10326,6 +10326,92 @@
   # pull: 5294 (PR title: Use FINE not FINEST for log messages that used to be INFO)
   # pull: 5295 (PR title: Update ATH)
 
+- version: '2.282'
+  date: 2021-03-02
+  changes:
+  - type: major bug
+    category: regression
+    pull: 5317
+    authors:
+    - olamy
+    references:
+      - pull: 5317
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.38.v20210224
+        title: Jetty 9.4.38 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.37.v20210219
+        title: Jetty 9.4.37 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.36.v20210114
+        title: Jetty 9.4.36 changelog
+    message: |-
+      Update Eclipse Jetty from 9.4.35.v20201120 to 9.4.38.v20210224 for bug fixes and enhancements.
+  - type: major rfe
+    category: major rfe
+    pull: 5049
+    issue: 64107
+    authors:
+    - kuisathaverat
+    - timja
+    references:
+      - issue: 55582
+      - issue: 64107
+      - pull: 5049
+      - url: https://github.com/jenkinsci/sshd-module
+        title: SSHD module repository
+    message: |-
+      SSHD module is no longer bundled in Jenkins core.
+      Provide SSHD as a plugin.
+  - type: rfe
+    category: rfe
+    pull: 5311
+    authors:
+    - basil
+    message: |-
+      Do not force plugin upgrades of recently detached plugins (regression in 2.279).
```

Comment:
Regression && RFE. Should it be a bug then? CC @timja who categorized it

Reply:
it wasn't really a bug I thought =/ there was a request to downgrade with justification so it was done.

but a bug was a stretch

Reply:
I removed the "regression in" text so that it won't confuse readers.